src/Methods/BuilderHelper.php

```diff
@@ -53,10 +55,15 @@ public function dynamicWhere(
         /** @var FunctionVariantWithPhpDocs $originalDynamicWhereVariant */
         $originalDynamicWhereVariant = ParametersAcceptorSelector::selectSingle($methodReflection->getVariants());
 
+        /** @var \PHPStan\Reflection\ParameterReflectionWithPhpDocs $originalParameter */
+        $originalParameter = $originalDynamicWhereVariant->getParameters()[1];
+
+        $actualParameter = new DummyParameter($originalParameter->getName(), new MixedType(), $originalParameter->isOptional(), $originalParameter->passedByReference(), $originalParameter->isVariadic(), $originalParameter->getDefaultValue());
+
         return new EloquentBuilderMethodReflection(
             $methodName,
             $classReflection,
-            [$originalDynamicWhereVariant->getParameters()[1]],
+            [$actualParameter],
             $returnObject
```

Comment:
```suggestion
            $returnObject,
            true
```

Laravel also allows to do something like `User::whereIdAndEmail(1, 'foo@example.com')->first()`

So, dynamic wheres can accept multiple arguments. If we can add this `true` here we can mark it as variadic and that should do the trick.

Reply:
Cool, wasn't aware that you could do dynamicWhere's with multiple  columns, fixed!